src/bci_build/package/keadhcp/README.md.j2

```diff
@@ -0,0 +1,33 @@
+# KEA DHCP Container
+
+This is an openSUSE Base Container Image for  DHCP server
```

Comment:
hardcoding openSUSE Base container image is not a good idea, because it is wrong for SLE. use the description placeholder instead.

Reply:
fixed